kymatio/scattering1d/conv_primitive.py

```diff
@@ -0,0 +1,27 @@
+import torch.nn as nn
```

Comment:
Maybe call it `convolution.py` for now or `convolution_1d.py` maybe. Do we duplicate filenames already? I think so. In that case use the first.